src/driver.coffee

```diff
@@ -51,11 +48,11 @@ class Neo4jDB
     else
       "#{value}"
 
-  reset: ->
-    console.log "Resetting Neo4j"
+  reset: () ->
+    console.log "Resetting Neo4j..."
+    @query "MATCH (n) OPTIONAL MATCH (n)-[r]-() DELETE n,r"
```

Comment:
Well, if you're hosting it on GrapheneDB, this is your best option. If you don't have any relationships, the version you suggest will not work. You need `OPTIONAL MATCH`.